WordPressKit/StatsServiceRemoteV2.swift

```diff
@@ -111,6 +158,48 @@ public protocol InsightProtocol {
     init?(jsonDictionary: [String: AnyObject])
 }
 
+// naming is hard.
+public protocol TimeStatsProtocol {
```

Comment:
`StatsPeriodProtocol`?

Reply:
I'll probably end up doing similar, but I desperately want to avoid using the word `Period` since it's _really confusing_ to my brain, given what was the last app I worked on.

It does fit perfectly though....

Reply:
Ah yeah, that makes total sense :) Maybe rename it when you've finished working on it?